CI/physmon/generate_comment.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+expected_reports = {""}
```

Comment:
Maybe add a comment whats the reasoning behind this? I think that would improve future maintainability...

Reply:
Ah this needs to go.

Reply:
```suggestion
```